octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -545,5 +603,53 @@ async def open_interest(self, feed, symbol, open_interest, timestamp, receipt_ti
     async def futures_index(self, **kwargs):
         pass
 
-    async def market_info(self, **kwargs):
-        pass  # Coingecko only
+    async def order(self, feed, symbol, data, receipt_timestamp):
+        """
+        Cryptofeed order callback
+        :param feed: the feed
+        :param symbol: the order symbol
+        :param data: a dict of miscellaneous data
+        :param receipt_timestamp: received timestamp
+        """
+        await self.push_to_channel(trading_constants.ORDERS_CHANNEL, orders=[self.exchange.parse_order(data)])
```

Comment:
exchange.parse_order is working here ? :o

Reply:
Yes. I only tested it for Bybit for now and it works :smiley:.